apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaApiClaimMessageMetaData.java

```diff
@@ -0,0 +1,94 @@
+package gov.cms.bfd.model.rda;
+
+import java.time.Instant;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@FieldNameConstants
+@Table(name = "`RdaApiClaimMessageMetaData`", schema = "`pre_adj`")
+public class RdaApiClaimMessageMetaData {
```

Comment:
can you put a javadoc on the class? (we are going to be turning on more javadoc checks pretty soon btw)

Reply:
Good point.  Comment has been added.